src/video/windows/SDL_windowswindow.c

```diff
@@ -236,14 +241,15 @@ static void WIN_AdjustWindowRect(SDL_Window *window, int *x, int *y, int *width,
     WIN_AdjustWindowRectWithStyle(window, style, menu, x, y, width, height, use_current);
 }
 
-void WIN_SetWindowPositionInternal(SDL_Window *window, UINT flags)
+int WIN_SetWindowPositionInternal(SDL_Window *window, UINT flags)
 {
     SDL_Window *child_window;
     SDL_WindowData *data = window->driverdata;
     HWND hwnd = data->hwnd;
     HWND top;
     int x, y;
     int w, h;
+    int return_variable = 0;
```

Comment:
This is typically named `result` in other SDL functions.

Reply:
Thanks. I used that one instead.